subprojects/soak/src/integTest/kotlin/org/gradle/kotlin/dsl/caching/ScriptCachingIntegrationTest.kt

```diff
@@ -234,19 +243,27 @@ class ScriptCachingIntegrationTest : AbstractScriptCachingIntegrationTest() {
         val settingsFile = cachedSettingsFile(withSettings(""), false, false)
         val buildFile = cachedBuildFile(withBuildScript("""task<MyTask>("myTask")"""), true)
 
+        // and: kotlin-dsl cache assertions
+        fun KotlinDslCacheFixture.assertCacheHits(run: Int) {
+            if (run == 1) {
```

Comment:
⭕️ How about adding a comment here about `buildSrc` not affecting the `settingsFile` classpath?

Reply:
Good point, done